cvat/apps/engine/ddln/tasks/vls_lines/models.py

```diff
@@ -137,6 +128,19 @@ def _calculate_vanishing_point(self, lines, reporter, is_lon):
         return average_point
 
 
+def _get_points_side(points, line):
+    # iterate points and figure out on which side of the line the points are placed
+    # returns -1 if on the left, 1 if on the left, 0 if all the points lie on the line
```

Comment:
so is it -1 or 1 when on the left?

Reply:
Copy-paste error, it's -1 when on the left.
I've changed the comment to be less confusing.